xbmc/utils/LangCodeExpander.cpp

```diff
@@ -234,6 +242,78 @@ bool CLangCodeExpander::ConvertToISO6392T(const std::string& strCharCode,
   return false;
 }
 
+bool CLangCodeExpander::LookupISO31661(std::string searchString,
+                                       std::string& convString,
+                                       bool returnCode)
+{
+  convString = "";
+
+  if (searchString.size() == 2) // Lookup based on ISO-3166-1 Alpha 2
```

Comment:
Please avoid magic numbers. Define and use a constant with a meaningful name. Here and everywhere else in this PR.

Reply:
Yes, for code maintainability reasons. 

> By definition 'ISO-3166-1 Alpha 2' is 2 bytes long

Maybe, but not every reader of the code has this knowledge. By putting a name to that number its purpose gets clear.

But please do not use a `#define`, in modern c++ you would use `static constexpr unsigned int`, which is type safe.